core/services/kraken/api/v1/routers/index.py

```diff
@@ -37,10 +37,19 @@ async def list_containers() -> Any:
 
 
 @index_router_v1.get("/log", status_code=status.HTTP_200_OK, response_class=PlainTextResponse)
-async def log_containers(container_name: str) -> StreamingResponse:
-    return StreamingResponse(
-        timeout_streamer(ContainerManager.get_container_log_by_name(container_name)), media_type="text/plain"
-    )
+async def log_containers(container_name: str, timeout: Optional[int] = None) -> StreamingResponse:
+    """
+    Fetch logs of a given container.
+    If timeout is provided, the stream will be closed after no log line is received for the given timeout.
+    If continuous is set to True, the stream will be kept open until the request be aborted by the client, even if the
+    container is stopped, it will keep trying to reconnect to the container log.
```

Comment:
There is no `continuous` parameter here.. is this docstring up to date??

Reply:
Already removed, it was an old parameter, thanks!